src/pytest_inmanta_lsm/lsm_project.py

```diff
@@ -896,12 +897,22 @@ def compile(
             self.project.compile(model, no_dedent=False)
             return
 
-        # Get the service targeted by the compile
-        service = self.get_service(service_id)
```

Comment:
I think we should srtill check that all the provided service ids are part of the inventory